### PR TITLE
update CDN references

### PR DESCRIPTION
**Pull request by pkra**
This PR changes the documents to reflect the loss of cdn.mathjax.org, and the suggested alternative of cdnjs.

### CODE REVIEW

**Comment by dpvc** [comment]
Looks good.

**Comment by pkra** [comment]
Thanks!

**Comment by pkra** [comment]
I'll do the other versions next week.

**Comment by pkra** [comment]
And the rest once we sort out the badges etc.
